presto-main/src/main/java/io/prestosql/sql/planner/plan/ApplyNode.java

```diff
@@ -62,6 +62,7 @@
      * <p>
```

Comment:
Can you please explain in the commit message what problem this commit solves?

Reply:
I tried reverting these commits in a branch of presto in qubole where the dynamic filtering feature is fully implemented, all the tests still passed.
Since I do not have context around why that change was necessary I've moved out those commits into a separate PR https://github.com/prestosql/presto/pull/462/